taichi/aot/module_data.h

```diff
@@ -33,11 +33,10 @@ struct CompiledFieldData {
 struct CompiledOffloadedTask {
   std::string type;
   std::string name;
-  // Do we need to inline the source code?
-  std::string source_path;
+  std::string source;
```

Comment:
Are we directly inlining the shaders now? What about SPIR-V?

Reply:
@k-ye good point! Hmmm I'll revert this back to source_path, and move writing to "spv/glsl" files to `convert()` instead.